Index: kernel-source/backports/drivers/net/wireless/ti/wlcore/Makefile
===================================================================
--- kernel-source.orig/backports/drivers/net/wireless/ti/wlcore/Makefile
+++ kernel-source/backports/drivers/net/wireless/ti/wlcore/Makefile
@@ -26,9 +26,13 @@ endef
 else
 # use git describe to determine version
 WLCORE_VERSION_DEPS := $(GIT_PATH).git/HEAD $(GIT_PATH).git/index $(GIT_PATH).git/refs/tags
-
+         
 define filechk_version.h
-	(echo 'static const char *wlcore_timestamp = __TIMESTAMP__;'; \
+	(echo '#if ((__GNUC__ <= 4)&&(__GNUC_MINOR__ < 9))';                                                           \
+         echo '  static const char *wlcore_timestamp = __TIMESTAMP__;';      \
+         echo '#else  ';                                                     \
+         echo '  static const char *wlcore_timestamp = "timestamp not possible with gcc 4.9";';\
+         echo '#endif ';                                                     \
 	echo 'static const char *wlcore_git_head = \
 			"$(shell $(GIT_CHDIR_CMD) git describe --dirty)";')
 endef
